js/client-teambuilder.js

```diff
@@ -168,9 +169,16 @@
 						formatText = '['+team.format+'] ';
 					}
 
-					buf += '<li><button name="edit" value="'+i+'" style="width:400px;vertical-align:middle">'+formatText+'<strong>'+Tools.escapeHTML(team.name)+'</strong><br /><small>';
+					buf += '<li><button name="edit" value="'+i+'" style="width:360px;vertical-align:middle">'+formatText+'<strong>'+Tools.escapeHTML(team.name)+'</strong><br /><small>';
 					buf += Storage.getTeamIcons(team);
-					buf += '</small></button> <button name="edit" value="'+i+'"><i class="icon-pencil"></i>Edit</button> <button name="delete" value="'+i+'"><i class="icon-trash"></i>Delete</button></li>';
+					if (this.currentMoving >= 0) {
+						buf += '</small></button> <button name="edit" value="'+i+'"><i class="icon-pencil"></i>Edit</button> ';
```

Comment:
That's because when a team is being moved the movement buttons are replaced by "Move below", "Move above" and "Keep here" buttons to choose the position to move the team and "Delete" buttons are suppressed. I don't know if it's the best way to do it.